pages/create.tsx

```diff
@@ -54,6 +62,71 @@ const Home: NextPage = () => {
     }
   }
 
+  interface NengajoTokenMetadata {
```

Comment:
Comment: 
(Line 60.)
In case of an error, it is better to show the Toast component.
=> Create a new issue.
https://chakra-ui.com/docs/components/toast

Reply:
I agree. 

It will be better to add this in a follow-up PR after moving the IPFS code to the `api` directory.